### PR TITLE
Renewal st

### PR DESCRIPTION
**Pull request by tombentley**
### Type of change

- Bugfix

### Description

This fixes `SecurityST.testCertificates()`. I think original test must have broken when @scholzj changed the zookeeper TLS sidecar to `verify = 2`, which meant that the sidecare couldn't verify the connection from `openssl s_client` (because the `s_client` didn't present a cert). I've fixed this by ensuring the connections to ZK use the Kafka broker cert (which is it bit of a hack, really, but works because they're also signed by the cluster CA).

I'm also piping `echo -n |` into the `openssl s_client` to terminate the connection after TLS negotiation. Without this each connection just hangs until timeout (10 minutes).

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Write tests
- [ ] Make sure all tests pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: systemtest/src/test/java/io/strimzi/systemtest/SecurityST.java [hunk 1]
```diff
@@ -24,35 +24,41 @@
 @ClusterOperator
 public class SecurityST extends AbstractST {
 
-    public static final String NAMESPACE = "security-cluster-test";
+    public static final String NAMESPACE = "myproject";
```
Comment (scholzj) [inline systemtest/src/test/java/io/strimzi/systemtest/SecurityST.java]: Is this change intentional? Are we now running the tests in the `myproject` namespace? (just curious)

Comment (tombentley) [reply systemtest/src/test/java/io/strimzi/systemtest/SecurityST.java]: Ah, damn. Thanks I will fix this.
